src/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.js

```diff
@@ -86,6 +91,18 @@ export default class geFormWidgetTokenizeCard extends LightningElement {
         }
     }
 
+    shouldHandleWidgetDataChange() {
+        return !this.isPaymentCharged();
+    }
+
+    isPaymentCharged() {
+        return (this.widgetDataFromState[apiNameFor(DATA_IMPORT_PAYMENT_STATUS_FIELD)] ===
```

Comment:
What do you think about naming these?

`let isCaptured = this.widgetDataFromState[apiNameFor(DATA_IMPORT_PAYMENT_STATUS_FIELD)] === this.PAYMENT_TRANSACTION_STATUS_ENUM.CAPTURED`
` let isSubmitted = ...`

and then

`return isCaptured || isSubmitted`